### PR TITLE
Ports: Enable readline for PHP & misc. improvements

### PR DESCRIPTION
**Pull request by GMTA**
* Ports `libiconv`, `readline` and `zlib` switched to SHA256 signature checks
* Outdated patch for `openssl` port removed
* Updated PHP to version 8.0.8 and enabled readline support

This allows for compilation of the PHP port on MacOS, and allows you to run `php` in interactive mode!

image

### CODE REVIEW

File: Ports/zlib/package.sh [hunk 1]
```diff
@@ -2,14 +2,11 @@
...
-https://www.zlib.net/zlib-${version}.tar.gz.asc zlib-${version}.tar.gz.asc"
-
-auth_type="sig"
-auth_import_key="783FCD8E58BCAFBA"
-auth_opts="zlib-${version}.tar.gz.asc"
+files="https://www.zlib.net/zlib-${version}.tar.gz zlib-${version}.tar.gz c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1"
+auth_type="sha256"
 
 configure() {
+    export CHOST=Linux
```
Comment (linusg) [inline Ports/zlib/package.sh:9]: Please add a comment to this explaining what's going on, like the commit message.

**Review by linusg — CHANGES_REQUESTED** [review]
MacOS => macOS in the 5th commit

File: Ports/zlib/package.sh (hunk 1 above)
Comment (GMTA) [reply Ports/zlib/package.sh]: Done!

**Comment by GMTA** [comment]
> MacOS => macOS in the 5th commit

Updated. Truly annoying way to capitalize that name, but it's the official spelling I guess :-)

**Review by linusg — APPROVED** [review]

**Pull request merged**